openquake/engine/aristotle.py

```diff
@@ -72,8 +72,7 @@ def get_close_mosaic_models(lon, lat, buffer_radius):
     hypo_buffer = hypocenter.buffer(buffer_radius)
     geoms = numpy.array([hypo_buffer])
     close_mosaic_models = geo.utils.geolocate_geometries(geoms, mosaic_df)
-    close_mosaic_models = sorted([model for model in close_mosaic_models[0]
-                                  if model != '???'])
+    close_mosaic_models = sorted([model for model in close_mosaic_models[0]])
```

Comment:
It is much better to write
```
[close_mosaic_models] = geo.utils.geolocate_geometries(geoms, mosaic_df)
```
(then it is clear that the returned list has a single element) and also move the `sorted` inside `geolocate_geometries`.

Reply:
Done